README.md

```diff
@@ -1 +1,164 @@
-# Enumerable-
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 My Enumerable ](#-my-enumerable-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 My Enumerable <a name="about-project"></a>
+
+**My Enumerable** a custom enumerable which have all?, any? and filter methods.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Enumerable module have all?, any? and filter method.
+- MyList class uses enumerable module 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Not deployed yet!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git SCM
+- Visual Studio Code
+- Live Server extension
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone https://github.com/biruk-bereka/Enumerable.git
+```
+
+Or download the ZIP folder.
+
+### Install
+
+Once you've cloned the repository simply run:
+
+```
+cd Enumerable
+```
+
+And this should open the directory with all the files from the repository on the command line, to open it in your explorer if you're using windows you can run:
+
+```
+explorer .
+```
+
+### Usage
+
+To run the project, if you are on the Enumerable folder path on your terminal you can execute the following command:
+
+```
+code .
+```
+
+This will open Visual Studio Code on the folder you're at.
+
```

Comment:
- [ ] [optional] @team great work on your documentation, it would have been better if you added how to run your project after cloning and opening it using vscode.